js/apps/admin-ui/src/authentication/form/CreateFlow.tsx

```diff
@@ -76,9 +76,15 @@ export default function CreateFlow() {
               }))}
             />
             <ActionGroup>
-              <Button data-testid="create" type="submit">
+              <FormSubmitButton
+                formState={formState}
+                data-testid="create"
+                type="submit"
```

Comment:
We can omit the `type` attribute, as the default is already set to the same value in `FormSubmitButton`.
```suggestion
```